modules/gdscript/tests/scripts/parser/features/single_line_class_statements.gd

```diff
@@ -0,0 +1,6 @@
+#GDTEST_OK
+
+pass;var x;@export;@onready;;;var y : float;pass
```

Comment:
```suggestion
pass;var x;@export;@warning_ignore("unused_private_class_variable");;;var _y : float;pass
```

1. Fixes error `"@onready" can only be used in classes that inherit "Node".`
2. Fixes warning `ONREADY_WITH_EXPORT`.